src/core/qgsrelationmanager.h

```diff
@@ -41,7 +41,7 @@ class CORE_EXPORT QgsRelationManager : public QObject
      * Constructor for QgsRelationManager.
      * \param project associated project (used to notify project of changes)
      */
-    explicit QgsRelationManager( QgsProject *project = nullptr );
+    explicit QgsRelationManager( QgsProject &project );
```

Comment:
Same considerations about API break and passing a pointer applies here.

Reply:
New commit that preserve API.